generators/angular/angular.js

```diff
@@ -0,0 +1,205 @@
+/**
+ * Main generator file for generating Angular projects.
+ */
+import * as fs from 'fs';
+import * as path from 'path';
+import options from './config/options';
+import questions from './config/questions';
+import { updateLog, errorLog } from '../../utils/log';
+import { execPromise } from '../../utils/exec';
+import { mapRouting } from './helpers/routing';
+import { deleteFolderSync, deleteFileSync } from '../../utils/delete';
+import { updatePackageJson, getlatestverion } from '../../utils/package';
+import { copyFolderRecursiveSync, copyFileSync } from '../../utils/copy';
+import HandlebarsTemplateGenerator from '../../utils/template-generator';
+
+function getQuestions() {
+  return questions;
+}
+
+function getOptions() {
+  return options;
+}
+
+class AngularAppGenerator {
+  /**
+   * @param configuration Configuration object
+   * @param execCallback Callback used for executing command on the child process spawn
+   * @param templateGenerator Instance of HandlebarsTemplateGenerator
+   */
+  constructor(configuration, execCallback, templateGenerator) {
+    this.execPromise = execCallback;
+    this.configuration = configuration;
+    this.generator = templateGenerator;
+  }
+
+  async start() {
+    updateLog('Preparing...');
+    await this.prepareDirectory();
+    await this.installAngular(this.configuration);
+    await this.installACPaasUI(this.configuration);
+    await this.createStarterTemplate(this.configuration);
+    updateLog('Done with front-end setup', 'cyan');
```

Comment:
```suggestion
    updateLog('Done with frontend setup', 'cyan');
```
Kind suggestion to be consistent in naming things.